### PR TITLE
New profile: prismlauncher

### PR DESCRIPTION
**Pull request by xplanthris**
Created a firejail profile for prismlauncher, I've attempted to limit it as much as possible while allowing Minecraft to run.
As for Downloads, PrismLauncher can get files from Downloads, I'm not sure for what so I have set it to read-only.
If anyone has any ideas on improvement let me know.

### CODE REVIEW

**Comment by xplanthris** [comment]
Also unrelated but if this does get merged, will it override my current profile and get everything running as if this was an in-built profile?

**Review by rusty-snake — CHANGES_REQUESTED** [review]
Please follow the profile template. It gives you a lot of suggestion for further hardening your prismlauncher profile and keeps profiles consistent.

File: etc/profile-m-z/prismlauncher.profile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+include /etc/firejail/whitelist-common.inc
+private-tmp
+private-dev
+disable-mnt
+blacklist /mnt
+blacklist /media
+blacklist /sbin
+
+# network
+net enp4s0f3u2u1u2
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:16]: The ne of the interface is a detail of *your* hardware.

**Comment by xplanthris** [comment]
> Please follow the profile template. It gives you a lot of suggestion for further hardening your prismlauncher profile and keeps profiles consistent.

Yeah apologies for that, I just used the Firejail configuration tool and modified it, will fix.

File: etc/profile-m-z/prismlauncher.profile (hunk 1 above)
Comment (xplanthris) [reply etc/profile-m-z/prismlauncher.profile]: I adjusted the profile to match the template

File: etc/profile-m-z/prismlauncher.profile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Firejail profile for PrismLauncher
+# Description: An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods.
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include PROFILE.local
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:6]:
```suggestion
include prismlauncher.local
```

File: etc/profile-m-z/prismlauncher.profile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Persistent global definitions
+include globals.local
+
+include allow-java.inc
+
+include disable-common.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-shell.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:15]: Every added `include disable-*.inc` is a win, you can try to add more.

File: etc/profile-m-z/prismlauncher.profile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+whitelist ${HOME}/.local/share/PrismLauncher
+whitelist ${HOME}/Downloads
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:18]: Every added `include whitelist-*-common.inc` is a win, you can try to add more.

File: etc/profile-m-z/prismlauncher.profile [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+nodvd
+nogroups
+nonewprivs
+noprinters
+noroot
+notpm
+notv
+nou2f
+protocol unix,inet,inet6
+seccomp
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:32]:
```suggestion
seccomp
seccoml.block-secondary
```

File: etc/profile-m-z/prismlauncher.profile [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+disable-mnt
+private-cache
+private-dev
+private-tmp
+
+dbus-system none
```
Comment (rusty-snake) [inline etc/profile-m-z/prismlauncher.profile:39]: Does `dbus-user none` work? If not, does
```
dbus-user filter
dbus-user.own org.prismlauncher.PrismLauncher
```
work?

**Review by rusty-snake — COMMENTED** [review]
Looks already good. Maybe we can harden it a bit more.

File: etc/profile-m-z/prismlauncher.profile (hunk 6 above)
Comment (xplanthris) [reply etc/profile-m-z/prismlauncher.profile:39]: dbus-user none works 100%, I tested the profile.

File: etc/profile-m-z/prismlauncher.profile (hunk 5 above)
Comment (xplanthris) [reply etc/profile-m-z/prismlauncher.profile:32]: Tested it now, I think you misspelled seccomp but anyways, I have committed this.

**Comment by xplanthris** [comment]
Thanks netblue30!

**Comment by netblue30** [comment]
no problem

**Pull request merged**